docs/pages/access-controls/reference.mdx

```diff
@@ -288,6 +290,12 @@ List of all rule options defined below.
     - resources:
       - auth_connector
       verbs:  [list, create, read, update, delete]
+    # SSH Sessions: Provides access to active sessions
+    # e.g If ssh sessions list is denied, users can't see the list of active sessions
+    # If the session read is denied, the active sesion list will show empty.
```

Comment:
I don't think this works as described because the `ssh_session` list/read privilege is assigned by default as part of implicit rules:
https://github.com/gravitational/teleport/blob/ab57eab5c059b323e4fb50cf02c1134745a19dd1/lib/services/role.go#L79

I've proposed to fix this situation in #8084.

Reply:
Actually, it _might_ work if the privilege is denied explicitly, instead of just being omitted from all roles assigned to a user.

Reply:
*Denying* `ssh_session` works. However, the note at the beginning
> Here's an example of an `allow` rule describing a `list` verb applied to the SSH `ssh_session` resource.  It means "allow users of this role to see a list of active SSH sessions".

can be misleading because, as it stands, the allow privilege is implicitly granted to every role irrespective of whether it is listed among the role's allow rules or not.